examples/tutorials/introduction/6_visualization.py

```diff
@@ -0,0 +1,167 @@
+"""
+Visualization
+=============
+
+Traits allows you to instantly create a graphical user interface
+for your ``HasTraits`` classes.  If you have TraitsUI and a suitable
+GUI toolkit (such as PyQt5 or PySide2) installed in your Python
+environment then you can create a dialog view of an instance of
+your class by simply doing::
+
+    >>> image.configure_traits()
+
+This gives you a default UI out of the box with no further effort,
+but it usually is not what you would want to provide for your users.
+
+With a little more effort using the features of TraitsUI, you can
+design a dialog which is more pleasing::
+
```

Comment:
I suggest adding a line with the corresponding TraitsUI imports above the class definition.